includes/utilities.inc

```diff
@@ -505,13 +505,13 @@ function islandora_solr_truncate_field_display($display_values, $max_length, $ad
 
     $updated_display_values
       = "<span class='toggle-wrapper'>" .
-      t("<span>!value !separator !show_more</span>", array(
+      t("<span>!value !show_more</span>", array(
         '!separator' => $separator,
```

Comment:
@jordandukart  good catch.
@whikloj this change isn't directly related to the new var, more of a bug related to separators. In the case of a comma separator, the last field value gets a comma appended to it. It's not actually separating values.

Eg: https://github.com/Islandora/islandora_xml_forms/blob/7.x/builder/transforms/mods_to_dc.xsl#L120 (terrible example, I know) if it's the last value, don't added a separator